### PR TITLE
[FIX] Timeout in SAM benchmark

### PR DESCRIPTION
**Pull request by eseiler**
Resolves https://github.com/seqan/product_backlog/issues/157

### CODE REVIEW

File: test/performance/io/format_sam_benchmark.cpp [hunk 1]
```diff
@@ -38,12 +27,12 @@ static std::string create_sam_file_string(size_t const n_queries)
...
-        size_t const seed = 1234u;
-        size_t const length_variance = 0;
-        size_t const reference_size = 500;
-        size_t const read_size = 100; // typical illumina read
-        std::string const default_query_id = "query_";
-        std::string const reference_id = "reference_id";
+        size_t const seed{1234};
+        size_t const length_variance{0};
+        size_t const reference_size{500};
+        size_t const read_size{100}; // typical illumina read
```
Comment (marehr) [inline test/performance/io/format_sam_benchmark.cpp:33]:
```suggestion
        size_t const seed{1234u};
        size_t const length_variance{0u};
        size_t const reference_size{500u};
        size_t const read_size{100u}; // typical illumina read
```

File: test/performance/io/format_sam_benchmark.cpp [hunk 2]
```diff
@@ -208,18 +197,26 @@ void seqan2_sam_file_read_from_disk(benchmark::State &state)
...
 #endif // SEQAN3_HAS_SEQAN2
 
-BENCHMARK(sam_file_read_from_stream)->Arg(50);
-BENCHMARK(sam_file_read_from_stream)->Arg(500);
+#ifndef NDEBUG
+static constexpr size_t low_query_count{5};
+static constexpr size_t high_query_count{10};
+#else
+static constexpr size_t low_query_count{50};
+static constexpr size_t high_query_count{500};
```
Comment (marehr) [inline test/performance/io/format_sam_benchmark.cpp:205]:
```suggestion
static constexpr size_t low_query_count{5u};
static constexpr size_t high_query_count{10u};
#else
static constexpr size_t low_query_count{50u};
static constexpr size_t high_query_count{500u};
```

File: test/performance/io/format_sam_benchmark.cpp [hunk 3]
```diff
@@ -5,23 +5,12 @@
...
 
-#include <seqan3/alignment/pairwise/alignment_result.hpp>
 #include <seqan3/alignment/pairwise/align_pairwise.hpp>
-#include <seqan3/alphabet/nucleotide/dna15.hpp>
-#include <seqan3/alphabet/quality/phred42.hpp>
 #include <seqan3/io/alignment_file/input.hpp>
-#include <seqan3/io/alignment_file/format_sam.hpp>
 #include <seqan3/io/alignment_file/output.hpp>
 #include <seqan3/test/performance/sequence_generator.hpp>
-#include <seqan3/test/seqan2.hpp>
```
Comment (smehringer) [inline test/performance/io/format_sam_benchmark.cpp:24]: wait. Why can we remove this while using `#if SEQAN3_HAS_SEQAN2` ?

Comment (eseiler) [reply test/performance/io/format_sam_benchmark.cpp:24]: `#include <seqan3/test/performance/sequence_generator.hpp>` includes it already

**Pull request merged**